docs/user-installation-disconnected.md

```diff
@@ -0,0 +1,62 @@
+# User Installation Guide - Disconnected Installations
+
+## Prerequisites/Requirements
+
+- Disconnected Mirror Image Registry created
+- Private cluster Installed: https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html
+- You created a registry on your mirror host: https://docs.openshift.com/container-platform/latest/installing/disconnected_install/installing-mirroring-installation-images.html#installing-mirroring-installation-images
+
+## Configuring the Disconnected Mirror Registry
+1. Extract the `openshift-tests` executable associated with the version of OpenShift you are installing.
+_Note:_ The pull secret must contain both your OpenShift pull secret as well credentials for the disconnected
+mirror registry.
+~~~
+PULL_SECRET=/path/to/pull-secret
+OPENSHIFT_TESTS_IMAGE=$(oc get is -n openshift tests -o=jsonpath='{.spec.tags[0].from.name}')
+oc image extract -a ${PULL_SECRET} "${OPENSHIFT_TESTS_IMAGE}" --file="/usr/bin/openshift-tests"
+chmod +x openshift-tests
+~~~
+
+2. Extract the images and the location to where they are to be mirrored from the `openshift-tests` executable.  
+
+~~~
+TARGET_REPO=target-registry.net/ocp-cert
+./openshift-tests images --to-repository ${TARGET_REPO} > images-to-mirror
+~~~
+
+3. Append Sonobuoy to the `images-to-mirror` list
+~~~
+SONOBUOY_TAG=$(./openshift-provider-cert-linux-amd64 version | grep "Sonobuoy Version:" | cut -d' ' -f 3)
+SONOBUOY_TARGET=${TARGET_REPO}/sonobuoy:${SONOBUOY_TAG}
+echo "quay.io/ocp-cert/sonobuoy:${SONOBUOY_TAG} ${SONOBUOY_TARGET}" >> images-to-mirror
+~~~
+
+4. Append the OPCT tools image to the `images-to-mirror` list
+
+~~~
+OPCT_VERSION=v0.3.0-alpha1
+OPCT_TARGET=${TARGET_REPO}/openshift-tests-provider-cert:${OPCT_VERSION}
+echo "quay.io/ocp-cert/openshift-tests-provider-cert:${OPCT_VERSION} ${OPCT_TARGET}" >> images-to-mirror
+~~~
+
+5. Mirror the images to the disconnected mirror registry
+
+~~~
+oc image mirror -a ${PULL_SECRET} -f images-to-mirror
+~~~
+
+
+## Preparing Your Cluster
+
+- The Insights operator must be disabled prior to to running tests.  See [Disabling insights operator](https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html)
+- The [Image Registry Operator](https://docs.openshift.com/container-platform/latest/registry/index.html) must be configured and available
+
+## Running Tests with the Disconnected Mirror registry
+
+To run tests such that they use images hosted by the Disconnected Mirror registry:
+
+~~~
+./openshift-provider-cert-linux-amd64 run --image-repository ${TARGET_REPO}
+~~~
+
+For additional details and configuration options, see [OpenShift Provider Certification Tool - User Guide](./user.md).
```

Comment:
@rvanderp3 sorry I missed that in my last review:
Our User Guide was structured sequentially. So, the section ["Usage/Run"](https://github.com/redhat-openshift-ecosystem/provider-certification-tool/blob/28d1cb070958d0f0d0d7c337819da41103e80058/docs/user.md#run-provider-certification-tests-) aims to describe each execution mode (regular, upgrade) and now disconnected. What do you think about moving this part (run) to that section?
Or feel free to suggest if it current way is confusing.